### PR TITLE
Maintenance/ember v3.28.5

### PR DESCRIPTION
**Pull request by Poltergeistz**
Ember cli version 3.28.5
Ember source 3.28.9
Ember data 3.28.10

- Removed the Webuniversum dependency for Appuniversum and changed the components accordingly.
- Linting.
- Prepared the app and dependencies to the ember LTS upgrade.
- Fixed some deprecation warnings except for ember-cli-babel and ember-simple-auth/torii.

Deprecation warnings are triggered by the login button :

- DEPRECATION: Using `run.later` has been deprecated. Instead, import the value directly from @ember/runloop: import { later } from '@ember/runloop'; 
- DEPRECATION: Use of `merge` has been deprecated. Please use `assign` instead. [deprecation id: ember-polyfills.deprecate-merge] See https://deprecations.emberjs.com/v3.x/#toc_ember-polyfills-deprecate-merge for more details.

### CODE REVIEW

File: app/styles/_shame.scss [hunk 1]
```diff
@@ -42,7 +42,7 @@
   position: absolute;
   right: 0;
   max-width: 480px;
-  top: $au-unit-large - $au-unit-tiny/2;
+  top: $au-unit-large - calc($au-unit-tiny / 2);
```
Comment (Windvis) [inline app/styles/_shame.scss:45]: Does this yield the expected result? Isn't `calc` a runtime css thing? I think you can simply multiply by `0.5` instead, or use the math.div thing that the deprecation message suggests(ed?).

File: app/routes/mock-login.js [hunk 2]
```diff
@@ -8,11 +8,12 @@ export default class MockLoginRoute extends Route {
     },
   };
 
+  @service router;
   @service session;
   @service store;
 
   beforeModel() {
-    if (this.session.isAuthenticated) this.transitionTo('index');
+    if (this.session.isAuthenticated) this.router.transitionTo('index');
```
Comment (Windvis) [inline app/routes/mock-login.js:16]: Unrelated to this PR, but might be a nice follow up:

I think modern ember-simple-auth has some util methods for this. In this case I think you could replace this with `prohibitAuthentication`: https://ember-simple-auth.com/api/classes/SessionService.html#method_prohibitAuthentication There is an equivalent for the inverse as well.

File: app/templates/components/data-table.hbs [hunk 3]
```diff
@@ -1,16 +1,23 @@
 <div data-table class="container-flex--scroll">
```
Comment (Windvis) [inline app/templates/components/data-table.hbs:1]: I think this file can simply be removed actually. I seems like the component isn't used in the app anymore?

File: app/styles/_shame.scss (hunk 1 above)
Comment (Windvis) [reply app/styles/_shame.scss:45]: Also, I think these styles are simply out-of-date in this project. The `.alert` class isn't used anymore and it currently looks like this:

image

So I think we can give improve that in some future PRs 😄. I guess the Loket app has a modernized setup.

**Review by Windvis — APPROVED** [review]
Thanks! I haven't ran it but I assume it's working. The comments I made are existing things that can be improved so we can do that in follow-up PRs I think.

File: app/styles/_shame.scss (hunk 1 above)
Comment (Poltergeistz) [reply app/styles/_shame.scss:45]: I can't remember, I will run the app tomorrow to see if the result is correct. Did not knew about math.div

File: app/templates/components/data-table.hbs (hunk 3 above)
Comment (Poltergeistz) [reply app/templates/components/data-table.hbs:1]: I think you're right.

**Comment by Windvis** [comment]
I ran into an issue with the new `@lblod/ember-submission-form-fields` release and with moment, so I pushed 2 commits to resolve those. The underlying issues will be fixed in future PR's.

**Comment by Windvis** [comment]
Thanks!

**Pull request merged**